src/ChannelServer/World/Map.cs

```diff
@@ -97,6 +134,11 @@ public void AddCharacter(Character character)
 
 			lock (_characters)
 				_characters[character.Handle] = character;
+
+			if (!SectorManager.Add(character, character.Position))
```

Comment:
Shouldn't it be enough to pass the entity? It has the Position property after all.